src/modules/UI/scenes/TransactionList/TransactionListConnector.js

```diff
@@ -56,6 +57,8 @@ const mapStateToProps = (state: State) => {
   const currencyInfo: EdgeCurrencyInfo = coreWallet.currencyInfo
   // set default requiredConfirmations to 1, so once the tx is in a block consider fully confirmed
   const requiredConfirmations = currencyInfo.requiredConfirmations ? currencyInfo.requiredConfirmations : 1
+  // eventually we will want the txList balance box visibility to be set to wallet-level setting (not account)
+  const isAccountBalanceVisible = state.ui.settings.isAccountBalanceVisible
```

Comment:
why are you using an account setting to control non account stuff?

Reply:
Because for now it **is** account level. @paullinator has specifically said that, since resources and time are limited, that we will go with Account-level specificity for the time being. I have already created a task in Asana for moving the TxList "Show Balance" feature to Wallet-level.

One step at a time.